spec/basket_spec.rb

```diff
@@ -2,19 +2,35 @@
 
 RSpec.describe Basket do
 
-  it "creates basket" do
-    expect Basket.new.is_a? Basket
+  let(:product1) { Product.new(name: "Football", price: 79) }
+  let(:product2) { Product.new(name: "Basketball", price: 60) }
+  let(:basket1) { Basket.new([product1, product2]) }
+
+  it "calculates products sum correctly" do
+    expect(basket1.sum).to eq 139
   end
 
-  let(:p1) { Product.new(name: "Football", price: 79) }
-  let(:p2) { Product.new(name: "Basketball", price: 60) }
   it "checks if sum with vat is correct" do
-    sum = Basket.new([p1, p2]).sum
-    sum_with_vat = Basket.new([p1, p2]).sum_with_vat
-    expect(sum * 1.23).to eq sum_with_vat
+    sum = Basket.new([product1, product2]).sum
+    sum_with_vat = Basket.new([product1, product2]).sum_with_vat
+    expect(sum * 1.23).to eq (sum_with_vat)
   end
```

Comment:
Well, you're right, I just had to read this test incorrectly but I would use `basket1` inside this `it "checks if sum with vat is correct" do` instead of creating another `Baske.new(..).sum` another time, you can do that for the `sum_with_vat` too.